docs/gen-docs/kyma_alpha_deploy.md

```diff
@@ -24,27 +24,28 @@ kyma alpha deploy [flags]
-  -k, --kustomization stringArray   Provides one or more kustomizations to deploy. 
-                                    Each flag occurrence accepts a URL with an optional reference (commit, branch, or release) in URL@ref format or a local path to the directory of the kustomization file.
-                                    By default, Lifecycle Manager is deployed from the GitHub main branch. (default [https://github.com/kyma-project/lifecycle-manager/config/default])
-      --kyma-cr string              Provide a custom Kyma CR file for the deployment.
-      --lifecycle-manager string    Installs Lifecycle Manager with the specified image:
-                                    - Use "my-registry.org/lifecycle-manager:my-tag"" to use a custom version of Lifecycle Manager.
-                                    - Use "europe-docker.pkg.dev/kyma-project/prod/lifecycle-manager@sha256:cb74b29cfe80c639c9ee9..." to use a custom version of Lifecycle Manager with a digest.
-                                    - Specify a tag to override the default one. For example, when specifying "v20230220-7b8e9515",  the "eu.gcr.io/kyma-project/lifecycle-manager:v20230220-7b8e9515" tag is used. (default "eu.gcr.io/kyma-project/lifecycle-manager:latest")
-  -m, --module stringArray          Provide one or more modules to activate after the deployment is finished. Example: "--module name@namespace" (namespace is optional).
-  -n, --namespace string            The Namespace to deploy the the Kyma custom resource in. (default "kyma-system")
-      --open-dashboard              Opens the Busola Dashboard at startup. Only works when a graphical interface is available and when running in interactive mode
-      --templates stringArray       Provide one or more module templates to deploy.
-                                    WARNING: This is a temporary flag for development and will be removed soon. (default [https://github.com/kyma-project/kyma/modules@main])
-  -t, --timeout duration            Maximum time for the deployment. (default 20m0s)
-      --wildcard-permissions        Creates a wildcard cluster-role to allow for easy local installation permissions of Lifecycle Manager.
-                                    Allows for Lifecycle Manager usage without worrying about modules requiring specific RBAC permissions.
-                                    WARNING: DO NOT USE ON PRODUCTIVE CLUSTERS! (default true)
+      --cert-manager string           Installs cert-manager from the specified static version. An empty string skips the installation. (default "v1.11.0")
+  -c, --channel string                Select which channel to deploy from. (default "regular")
+      --dry-run                       Renders the Kubernetes manifests without actually applying them.
+      --extra-templates stringArray   Provide one or more additional module templates via URL or local path to apply after deployment.
```

Comment:
```suggestion
      --extra-templates stringArray   Provides one or more additional module templates via URL or local path to apply after deployment.
```

Reply:
@IwonaLanger Are you sure about the tense here? The description is formulated as that the user has to provide one ore more module templates to use this flag. The flag itself does not actually provide module templates. How can we formulate it better?

Reply:
Same for the other flag descriptions that start with _Provide_

Reply:
Ah, I see. In that case my suggestions are irrelevant and everything is perfectly fine here. Sorry!

Reply:
If there's a possibility that the user might get it wrong (as I did), you could rephrase it like this: `You must provide...`. This will make a clear difference between a description and an instruction.